src/controller/src/clusters.rs

```diff
@@ -79,6 +79,8 @@ pub struct ReplicaAllocation {
     pub scale: u16,
     /// The number of worker threads in the replica.
     pub workers: usize,
+    /// The number of compute credits per hour.
+    pub compute_credits_per_hour: f64,
```

Comment:
It feels odd that this is a float (instead of e.g. an int representing 1000ths of a credit). I've traditionally been taught that anything accounting-related being a float is a code smell.

Reply:
What about `Numeric` (i.e. `Decimal<13>`), does that have the same code smell?

Reply:
AFAIK no

Reply:
I switched to `Numeric` then. I think it's a bit more straight forward than converting back and forth between 1000ths of a credit.